spring-addons-starter-rest/src/test/java/com/c4_soft/springaddons/rest/reactive/SpringAddonsServerWebClientSupportAnonymousTest.java

```diff
@@ -0,0 +1,54 @@
+package com.c4_soft.springaddons.rest.reactive;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.matching.AbsentPattern;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.wiremock.spring.EnableWireMock;
+import org.wiremock.spring.InjectWireMock;
+import reactor.test.StepVerifier;
+
+
+@ActiveProfiles("forward-bearer")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@EnableWireMock
+class SpringAddonsServerWebClientSupportAnonymousTest {
+
+    @Autowired
+    private WebClient test;
+
+    @InjectWireMock
+    private WireMockServer wireMockServer;
+
+    @Test
+    void forwardingBearerExchangeFilterFunctionTest() {
```

Comment:
The tests naming convention in this repo is inspired from BDD: `givenX_whenY_thenZ`. Here, it could be something like `givenAnEmptySecurityContext_whenSendRequestWithForwardingBearerExchangeFilterFunction_thenAuthorizationHeaderIsNotSet`.

It could be worth adding two more test cases:
```java
@Test
@WithAnonymousUser
void givenUserIsAnonymous_whenSendRequestWithForwardingBearerExchangeFilterFunction_thenAuthorizationHeaderIsNotSet() {
  ...
}
```
```java
@Test
@WithJwt
void givenUserIsAuthenticated_whenSendRequestWithForwardingBearerExchangeFilterFunction_thenAuthorizationHeaderIsSet() {
  ...
}
```